### PR TITLE
fix(route/bing): Replace `got`

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bing
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/bing/daily-wallpaper.ts [hunk 1]
```diff
@@ -16,25 +17,24 @@ export const route: Route = {
...
-    const data = response.data;
+    const data = response;
     return {
         title: 'Bing每日壁纸',
         link: 'https://cn.bing.com/',
         item: data.images.map((item) => ({
             title: item.copyright,
             description: `<img src="https://cn.bing.com${item.url}">`,
             link: item.copyrightlink,
+            pubDate: dayjs(item.startdate),
```
Comment (TonyRL) [inline lib/routes/bing/daily-wallpaper.ts:37]: Please use [`parseDate`](https://github.com/DIYgod/RSSHub/blob/dc093eb2e32513a5399176e041f201f393b6f1ee/lib/utils/parse-date.ts#L128) instead.

Comment (TonyRL) [inline lib/routes/bing/daily-wallpaper.ts:37]: Please use `.fullstartdate` instead

**Review by TonyRL — COMMENTED** [review]
Please update the route export as well
https://github.com/DIYgod/RSSHub/blob/1d72336ef33c7a4ce93cec35db6759678b2e3512/lib/routes/bing/daily-wallpaper.ts#L5-L17

**Comment by dzx-dzx** [comment]
> Please update the route export as well
> 
> https://github.com/DIYgod/RSSHub/blob/1d72336ef33c7a4ce93cec35db6759678b2e3512/lib/routes/bing/daily-wallpaper.ts#L5-L17

What exactly?

**Pull request merged**